releases/Process/Stabilization Process.md

```diff
@@ -0,0 +1,92 @@
+
+# Draft - O3DE Stabilization Process
+V1.0 Updated Jan 10, 2023. The release process is managed by sig-release. To request an update to this document, please open an issue at [https://github.com/o3de/sig-release/issues](https://github.com/o3de/sig-release/issues)
+																												
+
+O3DE Stabilization is the process we use to prepare the codebase for a release. During Stabilization we create a branch off of the development branch to create a “stabilization branch”, limit any new feature introduced into the branch, and the community fixes bugs. This helps to ensure O3DE releases do not ship with bugs that will block a user’s ability to utilize O3DE. Note that stabilization is not mean to catch every issue, and instead works in conjunction with unit tests, nightly automated test runs, functional automated testing, manually running test passes, and issues filed by the community.  
+  
+
+## Key Details
+
+-   The Stabilization Process typically takes place as part of the Release Process. Please refer to the [O3DE Major Release Process](https://github.com/o3de/sig-release/blob/main/releases/Process/Major%20Release%20Process.md) for more information about O3DE releases.
+-   As of Release 22.10.0, stabilization takes approximately 5 weeks from beginning stabilization to declaring the stabilization branch as "stable"
+-   Stabilization involves the following O3DE Repositories:
+    -   o3de
+    -   o3de-multiplayersample
+    -   o3de-netsoaktest
+
```

Comment:
Need to include the o3de-extras

Reply:
Why do we need to include extras for integration? Much like the Atom repos, keeping this up to date should be the responsibility of the owners of the gems and resources of sig-extras, not on sig release. As far as I am aware, everything in extras is 'extra' and not required to run core o3de.

Reply:
we will update the process once it is decided that O3DE extras is part of the release, which will happen after this discussion https://github.com/o3de/sig-release/discussions/129